### PR TITLE
Changes to check in and check out functionality

### PR DESCRIPTION
**Pull request by silentfatez**
For the check in and out html created the basic form for the 2 pages for the borrowing and returning page
For the Css file added basic style id tags for easier future edits 
for the javascript function added the code for the toggling between 2 html pages

### CODE REVIEW

File: assets/css/check-in-out.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#returnform {
+    background-color: lightblue;
+}
+#borrowform{
+  background-color: rgb(243, 178, 255)
```
Comment (mokjunneng) [inline assets/css/check-in-out.css]: need to include the ';' at the end

File: renderer-process/check-in-out/check-in.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+function show(p1,p2) {
+  document.getElementById(p1).style.display='block';
+  document.getElementById(p2).style.display='none';
+  return false;
+}
+// var page1=document.getElementById("Page1")
```
Comment (mokjunneng) [inline renderer-process/check-in-out/check-in.js]: Remove commented codes

File: sections/check-in-out/borrow-books.html [hunk 3]
```diff
@@ -1,5 +1,50 @@
 <template class="task-template">
+  <link rel="stylesheet" href="./assets/css/check-in-out.css">
```
Comment (mokjunneng) [inline sections/check-in-out/borrow-books.html]: put the link under index.html

File: sections/check-in-out/borrow-books.html [hunk 4]
```diff
@@ -1,5 +1,50 @@
...
   <section id="borrow-books-section" class="section js-section category-general">
-    <h1>Borrow books</h1>
+  <h1 id=borrowtitle>Borrow and return books </h1>
```
Comment (mokjunneng) [inline sections/check-in-out/borrow-books.html]: Check proper indentation
e.g. 
<grandparent>
    <parent>
        <child></child>
        <child></child>
    </parent>
    <parent></parent>
</grandparent>

**Comment by silentfatez** [comment]
Added changes as proposed by Mok

**Pull request merged**